app/models/subscription.rb

```diff
@@ -0,0 +1,10 @@
+# Model for mailing list subscriptions
+class Subscription < ApplicationRecord
+  validates :subscriber, presence: true
```

Comment:
Consider introducing a uniqueness validation to mitigate subscribers duplicating their subscription on the same list. See the `:scope` option for the uniqueness validator for more info.